library/components/TUrl.vue

```diff
@@ -25,18 +25,29 @@ export default {
     openInNewTab:{
       type: Boolean,
       default: true,
+    },
+    includeUrlStyle: {
+      type: Boolean,
+      default: true
     }
   },
   computed: {
     fullUrl() {
-      let allUrlParams = this.additionalUrlParams;
+      let allUrlParams = {...this.additionalUrlParams};
 
       if (this.includeFilterParams ) {
         const { 'context[org]': org, 'context[group]': group, ...restOfFilters } = this.getFiltersState;
```

Comment:
Non-blocking comment:
oh, interesting!! I added a getter to the global state to get context filters only. We could think about adding a getter to return all non context filters as well so that we can a centralized list of context filters we shouldn't touch: https://github.com/snyk/topcoat-core/blob/b0e61e137925e8427b3df162451d94823c688a25/frontend/src/store/modules/page.js#L85

Thought you might find it interesting, as we proceed adding components that reference context filters.

Reply:
That is a fantastic suggestion! I added a [Jira ticket ](https://snyksec.atlassian.net/browse/DP-326)so we don't forget. Wish that wouldn't be a whole seperate PR or I'd do it as part of this, but such is life.